components/pipeline-service/base/external-secrets/pipelines-as-code/kustomization.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - pipelines-as-code-secret.yaml
-namespace: pipelines-as-code
+namespace: openshift-pipelines
```

Comment:
yeah I think I need to temporarily duplicate components/pipeline-service/base/external-secrets/pipelines-as-code in both staging and prod, where I use the old namespace for pre 1.11, and the new namespace for post 1.11